### PR TITLE
Add header comments to Logic Test

### LINKED ISSUE #282: [Code Quality] Check and add comment for public class and methods
image

i noticed some classes have no headers comment, please check briefly and add them.

- [x] Kaiwei
- [x] Shao Cong
- [x] Zac
- [x] Jon
- [x] Wei Jun

Missing Headers (Help me edit once it is updated) :
seedu.trackermon.commons.core.JarTools
seedu.trackermon.commons.model.show.exceptions.DuplicatedShowException
seedu.trackermon.commons.model.show.exceptions.ShowNotFoundException
seedu.trackermon.commons.model.show.Comment
seedu.trackermon.commons.model.show.Name
seedu.trackermon.commons.model.show.Rating
seedu.trackermon.commons.model.show.Status
seedu.trackermon.commons.model.show.UniqueShowList
seedu.trackermon.commons.model.util.ReadOnlyShowList
seedu.trackermon.commons.model.util.ShowList
seedu.trackermon.commons.storage.JsonFileManager
seedu.trackermon.commons.ui.CommandBuilder

test:
seedu.trackermon.commons.core.index.IndexTest
seedu.trackermon.commons.core.ConfigTest
seedu.trackermon.commons.core.GuiSettingTest
seedu.trackermon.commons.core.JarToolsTest
seedu.trackermon.commons.core.VersionTest
seedu.trackermon.commons.util.AppUtilTest
seedu.trackermon.commons.util.CollectionUtilTest
seedu.trackermon.commons.util.ConfigUtilTest
seedu.trackermon.commons.util.FileUtilTest
seedu.trackermon.commons.util.StringUtilTest
seedu.trackermon.commons.model.show.NameTest
seedu.trackermon.commons.model.show.ShowTest
seedu.trackermon.commons.model.show.UniqueShowListTest
seedu.trackermon.commons.model.tag.TagTest
seedu.trackermon.commons.storage.JsonSerializableShowListTest
seedu.trackermon.commons.storage.JsonShowListStorageTest
seedu.trackermon.commons.storage.JsonUserPrefsStorageTest
seedu.trackermon.commons.storage.StorageManageTest
seedu.trackermon.commons.testutil.ShowBuilder
seedu.trackermon.commons.testutil.TypicalShows
seedu.trackermon.commons.ui.CommandBuilderTest
seedu.trackermon.commons.ui.UiPartTest
seedu.trackermon.commons.AppParametersTest

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/test/java/seedu/trackermon/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -76,13 +79,13 @@ public void execute_validCommand_success() throws Exception {
 
     @Test
```
Comment (JonathanHoshi) [inline src/test/java/seedu/trackermon/logic/LogicManagerTest.java:80]: Where is the header for the public test methods?

**Review by JonathanHoshi — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/trackermon/logic/LogicManagerTest.java (hunk 1 above)
Comment (lowkaiwei98) [reply src/test/java/seedu/trackermon/logic/LogicManagerTest.java:80]: This PR is only for header comments, public test methods include a lot of other test cases, i'll create a new PR for that

Comment (JonathanHoshi) [reply src/test/java/seedu/trackermon/logic/LogicManagerTest.java:80]: Noted

**Review by JonathanHoshi — APPROVED** [review]

**Review by arcornior — APPROVED** [review]
LGTM!

**Pull request merged**